collector/lib/SysdigService.h

```diff
@@ -89,6 +93,7 @@ class SysdigService : public Sysdig {
 
   void AddSignalHandler(std::unique_ptr<SignalHandler> signal_handler);
 
+  mutable std::mutex libsinsp_mutex_;
```

Comment:
Do we still need this mutex? I thought the point of using the workqueue is that we could directly access data from the scraper without blocking the main inpector thread.

Reply:
The purpose of this mutex is to avoid executing some Falco concurrently; which was very likely to happen with the previous approach, and a lot less now.
It still guards against executing CleanUp() (`inspector_close()`) while the Falco thread is inside GetNext()  (`inspector_.next()`) for instance.
However, this situation will likely never happen, while the mutex is in the hot path....

Reply:
A mutex that is not subject to a lot of contention does not cost much. So I would say 'yes'.